ignite/__init__.py

```diff
@@ -1,9 +1,9 @@
+__version__ = "0.5.0"
```

Comment:
I think this wont pass some code formatting tests as imports should be always on the top?

Reply:
ah ok. `__version__` is imported from ignite deeper in the tree so it causes a circular import. I need an alternative strategy then.